tokio/src/task/join_set.rs

```diff
@@ -362,7 +362,7 @@ impl<T: 'static> JoinSet<T> {
     /// This can happen if the [coop budget] is reached.
     ///
     /// [coop budget]: crate::task#cooperative-scheduling
-    fn poll_join_next(&mut self, cx: &mut Context<'_>) -> Poll<Option<Result<T, JoinError>>> {
+    pub fn poll_join_next(&mut self, cx: &mut Context<'_>) -> Poll<Option<Result<T, JoinError>>> {
```

Comment:
Can you enable `poll_join_next_with_id` too?
```rs
#[cfg(tokio_unstable)]
#[cfg_attr(docsrs, doc(cfg(tokio_unstable)))]
pub fn poll_join_next_with_id(
    &mut self,
    cx: &mut Context<'_>,
) -> Poll<Option<Result<(Id, T), JoinError>>> {
```

Reply:
Done 🙇‍♂️

Reply:
Please add the `cfg_attr` annotation too.